app/src/main/java/com/github/geohunt/app/model/database/api/Claim.kt

```diff
@@ -33,8 +33,16 @@ interface Claim {
      */
     val time: LocalDateTime
 
+    /**
+     * Distance between the actual location expressed in meter and the submitted one.
+     */
     val distance : Long
```

Comment:
Distance being a floating point value would make more sense.

Reply:
Did you see the comment i have made to the PR ?

Reply:
I mean it makes sense having points as an integer, but I don't see the connection with distances. Or maybe I missed a comment ?

Reply:
Oh yeah sorry so I CANNOT READ. Well it is expressed in meter which is roughly the precision of the GPS sensor. As such having a more precise unit is not very interesting. Furthermore this is only used to display the distance on the UI as such we don't really care if there are ~1/2 meter of error. Notice that this is what's in the database as such scores may use a different (and more precise) unit internally (cf. `long double`)

Reply:
I guess you're right it just didn't make sense to me to represent a distance as an integer but if it's only for display it should be fine.